firewood/src/v2/api.rs

```diff
@@ -69,12 +69,15 @@ pub enum Error {
     IncorrectRootHash { provided: HashKey, current: HashKey },
 
     /// Invalid range
-    #[error("Invalid range: {first_key:?} > {last_key:?}")]
+    #[error("Invalid range: {start_key:?} > {end_key:?}")]
     InvalidRange {
-        first_key: Vec<u8>,
-        last_key: Vec<u8>,
+        start_key: Box<[u8]>,
+        end_key: Box<[u8]>,
     },
 
+    #[error("limit must be positive")]
+    InvalidLimit,
```

Comment:
Is this used? Seems like the type system should prevent this

Reply:
removed